channeldb/invoices.go

```diff
@@ -189,7 +190,7 @@ type ContractTerm struct {
 	// PaymentPreimage is the preimage which is to be revealed in the
 	// occasion that an HTLC paying to the hash of this preimage is
 	// extended.
-	PaymentPreimage lntypes.Preimage
+	PaymentPreimage *lntypes.Preimage
```

Comment:
Does this mean we need to migrate over old instances that signal a hodl invoice with the `unknownPreimage` value?

Reply:
No, this is just the in-memory representation. On disk we'll still use `UnknownPreimage` (all zeroes)